### ISSUE TITLE
Refactor output writer

### ISSUE THREAD

**Issue by jhkennedy** [LIVVkit/LIVVkit#50]
With the refactor of LIVVkit elements to classes, the LIVVkit output should become a class containing all the output elements, and have methods to write the output for the different output representations (`HTML`, `JSON`, `LaTeX`, etc.). 

Also, this would allow for multiple outputs to be combined (see #49) easily as long as the JSON data is preserved --  every output should also include the JOSN data.

**Comment by jhkennedy** [comment]
All the LIVVkit elements have been refactored into classes and and have `_reper_*` methods to produce either HTML, JSON, or LaTeX via an appropriate jinja2 template. 

The way I'd suggest to resolve this issue finally is to: 
* require config files to have a `section`/`location`/`type` or whatever that specifies where inside the output report (website by default) the analysis' page should be located
* create a Report class that:
  * contains attributes describing the location of the report directory and other necessary info for the report (pretty much every global in `livvkit.__init__.py` 
  * creates the initial output structure based on the specs in the config files (before analyses are run!)
  * create a `Navigation` element that provides each pages navigation bar (this element also needs to be developed)
  * stores all the analyses' pages and summary elements
  * has a method to write the report after all the analyses are performed

Basically, the workflow should be (in `livv`) initialize a `Report`, create the initial structure as specified in each of the config files ( and a `imgs` sub directory?), then run the analyses adding the returned (report, summary) Elements into the report, and finally writing the output. 

This will completely eliminate the need for the associate JSON files in report directory and remove the JavaScript functions `drawNav`, `drawContent`, and `loadJSON` functions in `common.js`, and reduce `DrawContent` to:

```javascript
$(document).ready(function() {
$("#tabs").tabs();
});
```
just to make the tabs work. Almost entirely eliminating `common.js`. 

Also, the `self.Data` attribute in the `livvkit.elements.Page` initializer could then be eliminated.
